docs/source/getting-started/security-basics.rst

```diff
@@ -146,41 +184,69 @@ itself, ``jupyterhub_config.py``, as a binary string:
    If the cookie secret value changes for the Hub, all single-user notebook
    servers must also be restarted.
 
+.. _cookies:
 
-.. _authentication-token:
+Cookies used by JupyterHub authentication
+-----------------------------------------
 
-Proxy authentication token
---------------------------
+The following cookies are used by the Hub for handling user authentication.
 
-The Hub authenticates its requests to the Proxy using a secret token that
-the Hub and Proxy agree upon. The value of this string should be a random
-string (for example, generated by ``openssl rand -hex 32``).
+jupyterhub-hub-login
+~~~~~~~~~~~~~~~~~~~~
 
-Generating and storing token in the configuration file
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This is the login token used when visiting Hub-served pages encrypted such
```

Comment:
Why is "encrypted" here?

Reply:
I think it should be clearer if we say "Hub-served pages that are protected by authentication such...". Updating it now. Thanks!